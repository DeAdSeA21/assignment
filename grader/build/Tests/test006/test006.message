Output line 14:
  <i>expected: </i><span class='expected'>13</span>\t948.00
  <i>observed: </i>-<span class='observed'>-</span>- end of output ---
