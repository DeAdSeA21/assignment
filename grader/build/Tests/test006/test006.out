Starting loan balance: Annual interest rate: Monthly payment: 
Month	Balance
1	996.00
2	992.00
3	988.00
4	984.00
5	980.00
6	976.00
7	972.00
8	968.00
9	964.00
10	960.00
11	956.00
12	952.00
